lighthouse-core/test/fraggle-rock/config/config-test.js

```diff
@@ -133,10 +133,46 @@ describe('Fraggle Rock Config', () => {
       ],
     };
 
+    const {config} = initializeConfig(configJson, {gatherMode: 'snapshot'});
+    expect(config).toMatchObject({
+      artifacts: [{id: 'Accessibility', gatherer: {path: 'accessibility'}}],
+    });
+  });
+
+  it('should support artifact dependencies', () => {
+    const configJson = {
+      artifacts: [
+        {id: 'AppCacheManifest', gatherer: 'appcache'},
```

Comment:
no need to declare dependencies here if they're the default ones, it'll happen via symbol lookup